src/web/extractors.rs

```diff
@@ -55,11 +53,33 @@ impl DerefMut for DbConnection {
 /// as error response instead of a plain text "bad request"
 #[allow(clippy::disallowed_types)]
 mod path_impl {
+    use serde::de::DeserializeOwned;
+
     use super::*;
 
     #[derive(FromRequestParts)]
     #[from_request(via(axum::extract::Path), rejection(AxumNope))]
     pub(crate) struct Path<T>(pub T);
+
+    impl<T, S> OptionalFromRequestParts<S> for Path<T>
+    where
+        T: DeserializeOwned + Send + 'static,
+        S: Send + Sync,
+    {
+        type Rejection = AxumNope;
+
+        async fn from_request_parts(
+            parts: &mut Parts,
+            _state: &S,
+        ) -> Result<Option<Self>, Self::Rejection> {
+            <axum::extract::Path<T> as OptionalFromRequestParts<S>>::from_request_parts(
+                parts, _state,
+            )
```

Comment:
```suggestion
            parts.extract::<Option<axum::extract::Path<T>>>()
```

see https://docs.rs/axum/latest/axum/trait.RequestPartsExt.html#tymethod.extract :)

Reply:
nice! thanks